spec/services/migration/migrators/declaration_spec.rb

```diff
@@ -26,7 +26,7 @@ def setup_failure_state
       it "sets the created Declaration attributes correctly" do
         instance.call
         declaration = Declaration.find_by(ecf_id: ecf_resource1.id)
-        expect(declaration).to have_attributes(ecf_resource1.attributes.slice(:created_at, :udpated_at, :declaration_date, :declaration_type, :state))
+        expect(declaration).to have_attributes(ecf_resource1.attributes.slice("created_at", "udpated_at", "declaration_date", "declaration_type", "state"))
```

Comment:
this is now failing as we aren't setting the updated at correctly in the declaration migrator, could we look into that please